src/Helpers/JWKFetcher.php

```diff
@@ -38,39 +45,53 @@ public function __construct(CacheHandler $cache = null, $guzzleOptions = [])
     }
 
     /**
+     * Convert a certificate to PEM.
+     *
+     * @param string $cert Certificate to convert.
      *
-     * @param  string $cert
      * @return string
      */
     protected function convertCertToPem($cert)
     {
-        return '-----BEGIN CERTIFICATE-----'.PHP_EOL.chunk_split($cert, 64, PHP_EOL).'-----END CERTIFICATE-----'.PHP_EOL;
+        $output  = '-----BEGIN CERTIFICATE-----'.PHP_EOL;
+        $output .= chunk_split($cert, 64, PHP_EOL);
+        $output .= '-----END CERTIFICATE-----'.PHP_EOL;
+        return $output;
     }
 
     /**
+     * Fetch
      *
-     * @param string $iss
+     * @param string      $iss       Issuer for the JWKS being fetched.
+     * @param null|string $jwks_path Path to the JWKS from the issuer domain.
      *
      * @return array|mixed|null
-     *
-     * @throws \Exception
      */
-    public function fetchKeys($iss)
+    public function fetchKeys($iss, $jwks_path = null)
```

Comment:
I'd change this signature. In this method you're using the `$iss` just to prepend it to the `$jwks_path`. Why not just receive the full URL instead? You can go even further and let users specify a "local path" such as a location to a file included in the app but not hosted anywhere. 


Or similarly, you can rename the $iss to "host" and make the jwks_path have a default value of the default well-known path. Keeping the "issuer" mention on the docblock above would let users know how that "host" value must be.

Above suggestions are not breaking changes 🆗